govuk-one-login/authentication-api
# AUT-3906: Add cost allocation tags to authentication-api

## What

Add Cost Allocation tags to all components

I've updated the `default_tags` block on the providers to the following:
```
  default_tags {
    tags = {
      Environment = var.environment
      Owner       = "di-orchestration@digital.cabinet-office.gov.uk" / "di-authentication@digital.cabinet-office.gov.uk"
      Product     = "GOV.UK Sign In"
      System      = "${terraform-module-name}"

      application = "${terraform-module-name}"
    }
  }
```

This means that all the resources created by this terraform will now be correctly labeled and easily identified through the cost explorer, so we can better gauge what's costing what.

Note: `environment` => `Environment`. I would have left the two, but labels are case sensitive except for one specific instance, so it's easier to just keep one of these, otherwise we get flip-flopping resources.

## How to review

1. Deploy to a dev environment
2. Check some arbitrary resources have the expected tags
